README.adoc

```diff
@@ -136,8 +153,8 @@ include::finish/src/main/resources/META-INF/persistence.xml[tags=**;]
 
 The persistence unit is defined by the `<persistence-unit />` XML element. The `name` attribute is 
 required and is used to identify the persistent unit. The `transaction-type=”JTA”` element specifies 
-to use Java Transaction API (JTA) transactions. In a Java EE environment, the default is JTA if this 
-element is not specified.
+to use Java Transaction API (JTA) to manage the EntityManager's transactions. Since we are using
```

Comment:
Maybe something like:

> The 'transaction-type="JTA"' element specifies that the persistence context will enlist with Java Transaction API (JTA) global transactions.

Reply:
I say that because JPA will not do anything to control global transactions -- all it will do is either mark it for rollback, or respond to transaction callbacks (such as post-completion, etc.)  The only transactions that the JPA provider will directly control are the resource local transactions, those managed by EntityTransaction.

All that is just for your info, you don't need to mention that in the guide here.